### PR TITLE
Document the omnivector ubuntu PPA

### PR DESCRIPTION
**Pull request by heitorPB**
Docs for https://github.com/omnivector-solutions/slurm-charms/pull/115

Online at: https://omnivector-solutions.github.io/osd-documentation/testing/installation.html#deploy-slurm

### CODE REVIEW

File: installation.rst [hunk 1]
```diff
@@ -204,9 +204,9 @@ basic Slurm cluster:
   cloud environment. Currently available ones are AWS and LXD.
 - ``slurm-core/series/``: overlays to define the OS of the Slurm components,
   the possible options are CentOS7 and Ubuntu Focal (20.04).
-- ``slurm-core/charms/``: overlays to change the source of the charm. By
-  default, the bundle gets the ``latest/stable`` channel from Charmhub. The
-  overlays in this directory allows ut to change the source to either
+- ``slurm-core/charms/``: overlays to change the source of the charms. By
+  default, the bundle gets from the ``latest/stable`` channel from Charmhub.
```
Comment (dusktreader) [inline installation.rst:208]: This is a little unclear: "By default, the bundle gets from the ``latest/stable`` channel from Charmhub."

I'm not quite sure _what_ it's getting from that channel.

File: installation.rst [hunk 2]
```diff
@@ -204,9 +204,9 @@ basic Slurm cluster:
...
+  The overlays in this directory allows ut to change the source to either
```
Comment (dusktreader) [inline installation.rst:209]:
```suggestion
  The overlays in this directory allows us to change the source to either
```

File: installation.rst [hunk 3]
```diff
@@ -233,8 +233,18 @@ Juju will then download the charms from Charmhub, create the applications,
...
 will comprise the model.
 
+.. note::
+
+   The slurm-charms install SLURM from `Omnivector's OSD PPA
+   <https://launchpad.net/~omnivector/+archive/ubuntu/osd>`_ on Ubuntu. It is
+   possible to change the source to `Omnivector's Testing PPA
+   <https://launchpad.net/~omnivector/+archive/ubuntu/osd-testing>`_ (or to a
+   local cache server as well) whith the configuration ``custom-slurm-repo``.
+   Setting this value to repositories other than Omnivector's PPAs is not
```
Comment (dusktreader) [inline installation.rst:243]: Might be good to make this part bold or highlight it a bit

**Review by dusktreader — CHANGES_REQUESTED** [review]

File: installation.rst (hunk 1 above)
Comment (heitorPB) [reply installation.rst]: Is it clear now?

File: installation.rst (hunk 3 above)
Comment (heitorPB) [reply installation.rst:243]: It already is! Just pushed to `testing`: https://omnivector-solutions.github.io/osd-documentation/testing/installation.html#deploy-slurm

**Review by dusktreader — APPROVED** [review]

**Pull request merged**